pkg/config/config.go

```diff
@@ -188,6 +190,9 @@ type Workflow interface {
 	FromAddress() *types.EIP55Address
 	ForwarderAddress() *types.EIP55Address
 	GasLimitDefault() *uint64
+	TxAcceptanceState() *commontypes.TransactionStatus
+	PollPeriod() *commonconfig.Duration
+	AcceptanceTimeout() *commonconfig.Duration
```

Comment:
Life is much easier with std durations:
```suggestion
	PollPeriod() time.Duration
	AcceptanceTimeout() time.Duration
```
The config types are specialized for parsing, but don't need to be exposed through the config interfaces (though pointer is OK case by case).